### PR TITLE
script to analyse and delete old seqr_loader folders in seqr-main

### PR DESCRIPTION
**Pull request by EddieLF**
This script looks through the genome folders in `cpg-seqr-main/seqr_loader`, and then separately the exome folders in 
`cpg-seqr-main/exome/seqr_loader`.

The idea is to search for the `JointGenotyping/full.vcf.gz` file and check the datetime created. We collect all these timestamps and remove the latest entry, as well as any entries younger than 30 days. 
For the remaining entries (i.e. those 30+ days old) we want to delete the entire directory associated with the VCF. 

There are a few folders that contain no VCF, just some parts files or interval files. For these, I'm less sure about deleting them so for now I am just printing them out with the logger. I imagine we do want to delete these if they are older but we can start with the folders containing VCFs for now.

### CODE REVIEW

File: data_transfer/cleanup_seqr_loader_files.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    seqr_load_directories = set()
+    seqr_loads = {}
+    for blob in bucket.list_blobs(prefix=prefix):
+        if '.mt' in blob.name or '.ht' in blob.name:
+            continue
+
+        # Some directories have no VCF, but we still want them
+        if blob.name.removeprefix(prefix).split('/')[0] not in seqr_load_directories:
+            seqr_load_directories.add(blob.name.removeprefix(prefix).split('/')[0])
```
Comment (MattWellie) [inline data_transfer/cleanup_seqr_loader_files.py]:
```suggestion
        if (seqr_dir := blob.name.removeprefix(prefix).split('/')[0]) not in seqr_load_directories:
            seqr_load_directories.add(seqr_dir)
```
WALRUS!

**Review by MattWellie — COMMENTED** [review]
Looks good!

File: data_transfer/cleanup_seqr_loader_files.py (hunk 1 above)
Comment (EddieLF) [reply data_transfer/cleanup_seqr_loader_files.py]: Goo goo gjoob!

**Review by cassimons — APPROVED** [review]
Looks good thanks 👍

**Pull request merged**